egs/ljspeech/tts1/local/update_json.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+
+verbose=0
+
+. utils/parse_options.sh
+set -e
+set -u
+
+if [ $# != 2 ]; then
+    echo "Usage: $0 <json> <scp>";
+    exit 1;
+fi
+
+json=$1
+feat=$2
+dir=$(dirname ${json})
+tmpdir=`mktemp -d ${dir}/tmp-XXXXX`
+rm -f ${tmpdir}/*.scp
+
+# feats scp
+cat ${feat} > ${tmpdir}/feat.scp
+
+# ilen & idim scp
+feat-to-len scp:${feat} ark,t:${tmpdir}/ilen.scp
```

Comment:
Try to avoid to use `feat-to-len`. See https://github.com/espnet/espnet/blob/master/src/utils/data2json.sh#L30-L32